src/components/Admin/index.jsx

```diff
@@ -51,14 +42,31 @@ class Admin extends React.Component {
   }
 
   renderErrorMessage() {
-    const message = `There was a problem fetching dashboard data: ${this.props.error.message}`;
-    return <StatusMessage alertType="danger" message={message} />;
+    return (
+      <StatusAlert
```

Comment:
Lets wrap all of this into a ErrorAlert component that just takes the message as props so we can reuse it here and in CourseEnrollments

Reply:
@georgebabey The `StatusAlert` here is no longer the Paragon `StatusAlert`, but rather the new wrapped component in `components/StatusAlert` that is reused here and in `CourseEnrollments`.